src/app/style.scss

```diff
@@ -0,0 +1,24 @@
+
+.container {
+	flex: 1;
+	justify-content: flex-start;
+	background-color: #fff;
+}
+
+.blockHolderSemiBordered {
```

Comment:
With the BlockHolder now migrated to the Gutenberg side (by utilizing the BlockEdit component), it's weird that there's a style in the mobile repo about it. Can we move it somehow to the Gutenberg side?

I know that this style is used for the post title component too but, this might be the opportunity to reconcile the two?

Reply:
I'll see if I can reconcile those 👍
Again I went the easy way first for this PR, basically just moving modules around and porting the syntax whenever necessary. I agree there is room for improvement/refactoring but I wonder if we should address those beforehand in this PR or merge rapidly to avoid conflicts and refactor after?

Reply:
I think it's OK to tackle this one in a separate PR 👍.

Let's add it to a "Known issues" type of section in this PR description so we can help us keep track of it, wdyt?